infrastructure/constructs/processing.py

```diff
@@ -82,7 +83,7 @@ def __init__(
         datasets_table: Table,
         git_commit_parameter: aws_ssm.StringParameter,
     ) -> None:
-        # pylint: disable=too-many-locals, too-many-statements
+        # pylint: disable=too-many-locals, too-many-statements, too-complex
```

Comment:
Looks like this method should be pulled apart. Would you be OK to do that in a separate branch?

Reply:
Yes I was thinking the same!